project-words.txt

```diff
@@ -473,3 +473,12 @@ REFID
 unroute
 myown
 yass
+becasue
```

Comment:
Can we fix the spelling errors like `becasue: because`, `Subsciption: subscription`, and `wihtout: without`? The rest are all fine

Reply:
Fixed.